front/src/StandardButton.test.js

```diff
@@ -0,0 +1,23 @@
+import React from "react";
+import SRRButton from "./StandardButton";
+import renderer from "react-test-renderer";
+
+const testFunc = () => {
+  return "Yeah, that'll do.";
+};
+
+describe("Button can be clicked", () => {
```

Comment:
`describe` block is used to organize assertions that all pertain to the same feature or context. In this case, you test the `SRRButton` component, so pass `"SRRButton"` to the `describe` block.

Reply:
that's just the testfunction? or do you mean I should remove the "it"?